bin/vagrant

```diff
@@ -78,6 +78,12 @@ if argv.include?("--debug-timestamp")
   ENV["VAGRANT_LOG_TIMESTAMP"] = "1"
 end
 
+# Setting to enable/disable showing progress bars
+if argv.include?("--quiet-progress")
```

Comment:
I don't know about this
- Seems like this is the pattern set for `--debug` and `--timestamp`. I think `--quiet-progress` is in the same class of options as those
- Any other suggestions about naming this option? I think this makes sense since it is not silencing the whole output. But also, I think this name is pretty clunky. I thought against `--quiet` since this options doesn't truly quiet all the output but I'm happy to rethink that

Reply:
To your second point, I agree that just `--quiet` will be a bit misleading if we're using this flag to disable progress output. I like `--quiet-progress`, I think it's short and descriptive enough that hopefully folks know what it's for. Unfortunately if we define this flag globally like this, we won't be able to show it easily in any CLI help output, so having it be named something short and descriptive is ideal! And because of that, we'll want to make sure to update the documentation with this flag and the corresponding environment variable.